docs/team/yleeyilin.md

```diff
@@ -0,0 +1,46 @@
+---
+layout: page
+title: Yi Lin's Project Portfolio Page
+---
+
+### Project: AddressBook Level 3
```

Comment:
Should we make this project description specific to our AB3 aka poochplanner

Reply:
Agreed, ill make the required changes.